### PR TITLE
feat(sms): Add SMS plugin

### PR DESCRIPTION
**Pull request by kellyjandrews**
This PR adds the plugin for sending SMS with the CLI.

In order to test, have a working local copy of the cli, and use `bin/run plugins:link ./packages/sms` and run tests with `npx lerna run test --scope=`@vonage/cli-plugin-sms`

### CODE REVIEW

File: packages/sms/src/commands/sms/index.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import BaseCommand from '@vonage/cli-utils';
+import { request } from '@vonage/vetch'
+import { flags, OutputFlags } from '@oclif/parser';;
```
Comment (amdcavallaro) [inline packages/sms/src/commands/sms/index.ts:3]:
```suggestion
import { flags, OutputFlags } from '@oclif/parser';
```

File: packages/sms/src/commands/sms/index.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import BaseCommand from '@vonage/cli-utils';
+import { request } from '@vonage/vetch'
```
Comment (amdcavallaro) [inline packages/sms/src/commands/sms/index.ts:2]: Hey @kellyjandrews I notice some lines end in ; others not. Maybe keep it consistent across the files?
```suggestion
import { request } from '@vonage/vetch';
```

Comment (kellyjandrews) [reply packages/sms/src/commands/sms/index.ts:2]: Need to get eslint set up properly so it will keep it consistent, yes.

**Review by matt-lethargic — APPROVED** [review]

File: packages/sms/src/commands/sms/index.ts [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  to: any,
+  from: any,
+  message?: any
+}
+
+export default class SMSSend extends BaseCommand {
+  static description = 'Send a simple SMS.'
+
+  static examples = [
+    `vonage sms --to=5551234567 --from=5551234567 --message="Hello there!`,
```
Comment (pardel) [inline packages/sms/src/commands/sms/index.ts:15]: number format used in example is invalid

File: packages/sms/package.json [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    "files": [
+        "/dist",
+        "/bin",
+        "/npm-shrinkwrap.json",
+        "/node_modules",
+        "/oclif.manifest.json"
+    ],
+    "homepage": "https://github.com/Vonage/vonage-cli",
+    "keywords": [
+        "oclif-plugin"
```
Comment (pardel) [inline packages/sms/package.json:44]: can we add Vonage, CLI & this plugin specific keywords?

**Review by pardel — CHANGES_REQUESTED** [review]

**Review by pardel — CHANGES_REQUESTED** [review]
sorry, would like to request some changes

File: packages/sms/README.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
+@vonage/cli-plugin-numbers
```
Comment (pardel) [inline packages/sms/README.md:1]: Seems like the wrong info here

File: packages/sms/oclif.manifest.json [hunk 6]
```diff
@@ -0,0 +1 @@
+{"version":"1.1.0","commands":{"sms":{"id":"sms","description":"","pluginName":"@vonage/cli-plugin-numbers","pluginType":"core","aliases":[],"examples":["vonage sms"],"flags":{"help":{"name":"help","type":"boolean","char":"h","description":"show CLI help","allowNo":false},"apiKey":{"name":"apiKey","type":"option","hidden":true},"apiSecret":{"name":"apiSecret","type":"option","hidden":true},"appId":{"name":"appId","type":"option","hidden":true},"keyFile":{"name":"keyFile","type":"option","hidden":true},"trace":{"name":"trace","type":"boolean","hidden":true,"allowNo":false}},"args":[]}}}
```
Comment (pardel) [inline packages/sms/oclif.manifest.json:1]: pluginName is wrong here

File: packages/sms/src/commands/sms/index.ts [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+  to: any,
+  from: any,
+  message?: any
+}
+
+export default class SMSSend extends BaseCommand {
+  static description = 'Send a simple SMS.'
+
+  static examples = [
+    `vonage sms --to=15551234567 --from=15551234567 --message="Hello there!`,
```
Comment (pardel) [inline packages/sms/src/commands/sms/index.ts:15]: There is a missing quote at the end of the example - we should also used single-quotes. The double don't work on the macOS default terminal

File: packages/sms/oclif.manifest.json (hunk 6 above)
Comment (pardel) [reply packages/sms/oclif.manifest.json:1]: The help command doesn't seem to be implemented with an example:
Screenshot 2022-02-10 at 22 10 39

**Review by pardel — CHANGES_REQUESTED** [review]

File: packages/sms/src/commands/sms/index.ts (hunk 7 above)
Comment (kellyjandrews) [reply packages/sms/src/commands/sms/index.ts:15]: Good catch - thanks! I'll update that this afternoon.

File: packages/sms/oclif.manifest.json (hunk 6 above)
Comment (kellyjandrews) [reply packages/sms/oclif.manifest.json:1]: Right - this I need to investigate - see https://jira.vonage.com/browse/DEVX-5881

Seems to only be the SMS plugin, so not exactly sure what I've missed.

File: packages/sms/README.md (hunk 5 above)
Comment (kellyjandrews) [reply packages/sms/README.md:1]: 🤦🏻‍♂️ thanks - definitely need to change that

**Review by abdulajet — APPROVED** [review]
lgtm

**Comment by kellyjandrews** [comment]
@pardel Can you also approve - I'd like to get these changes published. Thanks!

**Pull request merged**